### ISSUE TITLE
Output childs of the named class

### ISSUE THREAD

**Issue by anilkumarmyla** [mikeyhu/scaladiagrams#9]
Current `--parent` option outputs parents of the named class. It's great to have the opposite as well, given a named class, output all children.

**Comment by mikeyhu** [comment]
Yeah, does sound like it could be useful. I'll take a look and see if how doable it is.
